flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -832,14 +833,42 @@ private StateBackend loadStateBackend(ReadableConfig configuration, ClassLoader
 	 * @param to
 	 * 		The number to stop at (inclusive)
 	 * @return A data stream, containing all number in the [from, to] interval
+	 * @deprecated Use {@link #fromSequence(long, long)} instead to create a new data stream
+	 * that contains {@link org.apache.flink.api.connector.source.lib.NumberSequenceSource}.
```

Comment:
We shouldn't mention the internal implementation here. 

```suggestion
	 * @deprecated Use {@link #fromSequence(long, long)} instead to create a new sequence data stream.
```